### ISSUE TITLE
[Bug]: Hey team, I use to use "facebook_app_events:     git:       url: https://github.com/oddbit/flutter_facebook_app_events       ref: fix/iOS-issue-0.19.4" in pubspec.yaml but now build is failing as I suspect the branch has been merged and removed? Would you be kind enough to suggest what should I replace it with (sorry first Flutterflow project so very new)

### ISSUE THREAD

**Issue by AmorApp** [oddbit/flutter_facebook_app_events#387]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Builds would work for ios using...
facebook_app_events:
    git:
      url: https://github.com/oddbit/flutter_facebook_app_events
      ref: fix/iOS-issue-0.19.4
      
    But now it's failing again on...
    Xcode archive done.                                         139.1s
Failed to build iOS app
Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_Builtin_float

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_errno

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_math

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_signal

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_stdio

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swift_time

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swiftsys_time

Error (Xcode): Undefined symbol: __swift_FORCE_LOAD_$_swiftunistd

Error (Xcode): Linker command failed with exit code 1 (use -v to see invocation)

Encountered error while archiving for device.

Build failed :|
Step 11 script `Flutter build ipa and automatic versioning` exited with status code 1

### Expected Behavior

Build should publish with the workaround of the git work you've done

### Steps To Reproduce

I have also tried the following but no luck.

environment:
  sdk: ">=3.0.0 <4.0.0"

dependencies:
  facebook_app_events:
    git:
      url: https://github.com/oddbit/flutter_facebook_app_events
      ref: develop  # Refers to the 'develop' branch

### Platform

iOS

### Flutter Doctor

```shell
I'm super sorry, I'm new and struggling to use Xcode so just working directly with flutter flow and using your code for Facebook events
```

**Comment by antonoddbit** [comment]
@AmorApp it's merged to main, please see main branch in 0.19.4 version

**Comment by AmorApp** [comment]
> @AmorApp it's merged to main, please see main branch in 0.19.4 version

Thanks heaps Anton! Sorry to be a pain, I've done the following, but still getting the swift error, is it the info below that needs editing or other parts of github also? Sorry, I'm a beginner.
  facebook_app_events:
    git:
      url: https://github.com/oddbit/flutter_facebook_app_events
      ref: main

**Comment by AmorApp** [comment]
Hey @antonoddbit - so just to confirm, should it now be the following, with the other changes from (https://github.com/oddbit/flutter_facebook_app_events?files=1) ReadMe

facebook_app_events:
git:
url: https://github.com/oddbit/flutter_facebook_app_events
ref: 3e4e062ded3eb92b37e43c274ee84da4a21edfa2

**Comment by antonoddbit** [comment]
Hi @AmorApp, previously you are using the draft that used Audience network in version 6.15.0, but it merged in latest update to using 6.16.0. That ref is pointing to commit that still using version 6.15.0
